rail_scripts/rail-preprocess-parquet

```diff
@@ -23,11 +23,17 @@ Options:
                                       [default: mag_{band}].
     --mag-error-template=<template>   Set the names of the magnitude error
                                       columns [default: magerr_{band}].
+    --ra=<ra>                         Set the name of equatorial coordinate
+                                      Right Ascension [default: coord_ra]
+    --dec=<dec>                       Set the name of equatorial coordinate
+                                      Declination [default: coord_dec]
 
 
 The following preprocessing steps are currently implemented:
 - Limit number of rows per file to a maximum, splitting the file if necessary.
-- Generate files only with magnitude columns.
+- Generate files only with object id, coordinates (optional, if available), 
+  magnitudes and error columns.
+- Object must be part of input files as indexes.
```

Comment:
This rule is not a requirement of this script, it will work without this. Maybe that's a rule of some other further script ?